src/query/api/v1/handler/prometheus/remote/write.go

```diff
@@ -33,7 +33,7 @@ import (
 	"github.com/m3db/m3/src/query/storage"
 	"github.com/m3db/m3/src/query/ts"
 	"github.com/m3db/m3/src/query/util/logging"
-	"github.com/m3db/m3/src/x/net/http"
+	xhttp "github.com/m3db/m3/src/x/net/http"
```

Comment:
My version of vs code updates automatically - I'll figure out how to remove unless no one cares.